### ISSUE TITLE
Syntax error swallowed by incorrectly localized message.

### ISSUE THREAD

**Issue by kevcunnane** [microsoft/azuredatastudio#5597]
Issue Type: <b>Bug</b>

This is a syntax error but the error returned is all messed up
 Reported by an internal user: run 
```
CREATE EXTERNAL DATA SOURCE SqlDataPool WITH(LOCATION = 'sqldatapool://controller-svc:8080/datapools/default
 ```
Get the following:
```
Execution failed due to an unexpected error: 
{1}
Total execution time: 00:00:00
```
Expected:
- Get actual string instead of {1}.

Azure Data Studio version: Azure Data Studio 1.8.0-insider (2ca39b571a91832a8c2fa4a26726c29f31292841, 2019-05-22T07:15:07.151Z)
OS version: Darwin x64 18.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.09GB free)|
|Process Argv|-psn_0_188462|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (4)</summary>

Extension|Author (truncated)|Version
---|---|---
azdata-sanddance|dan|0.2.290
newdatabase|kev|0.0.3
azuredatastudio-postgresql|Mic|0.1.0
sql-vnext|Mic|0.12.1

</details>

**Comment by Charles-Gagnon** [comment]
Fixed with #5577 

If the user wants to update to latest Insider build they should see the actual error :

![image](https://user-images.githubusercontent.com/28519865/58290171-966ace00-7d6d-11e9-8ef2-255ff7fec9cb.png)

**Comment by kevcunnane** [comment]
Thanks Charles!
